### PR TITLE
Exception commits

### PR DESCRIPTION
**Pull request by Exception-UA**

### CODE REVIEW

File: history/units/RSE_3236.txt [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+}
+
+units = {
+	division = {
+		location = 468
+		division_template = "Marines Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
```
Comment (Cleverking2003) [inline history/units/RSE_3236.txt:95]: One of armored brigades is extra

File: history/units/TRA_3236.txt [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 2021
+		division_template = "Infantry Brigade with MRLS"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
```
Comment (Cleverking2003) [inline history/units/TRA_3236.txt:327]: Two extra infantry and two missing artillery. Should these be the "Infantry with MRLS"?

**Review by Cleverking2003 — CHANGES_REQUESTED** [review]

File: history/units/TRA_3236.txt (hunk 2 above)
Comment (Exception-UA) [reply history/units/TRA_3236.txt:327]: Yes, I w/ MLRS = artillery

File: history/units/RSE_3236.txt (hunk 1 above)
Comment (Exception-UA) [reply history/units/RSE_3236.txt:95]: It's not. Common trucks are not armored

File: history/units/KAE_3236.txt [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+	division = {
+		location = 2631
+		division_template = "Infantry Brigade"
+		start_experience_factor = 0.15
+		start_equipment_factor = 0.8
+	}
+}
```
[the remark this answers is not in the document]
Comment (Exception-UA) [reply history/units/KAE_3236.txt:84]: Added

File: history/units/GKR_3236.txt [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+		start_equipment_factor = 0.9
+		start_experience_factor = 0.4
+	}
+	division = {
+		location = 1388
+		division_template = "Division de Infanteria"
+		start_equipment_factor = 0.9
+		start_experience_factor = 0.4
+	}
+}
```
[the remark this answers is not in the document]
Comment (Exception-UA) [reply history/units/GKR_3236.txt:390]: Added

**Pull request merged**